core/network/network.go

```diff
@@ -75,6 +75,20 @@ func (s IDSet) Contains(id Id) bool {
 	return exists
 }
 
+// Difference returns a new IDSet representing all the values in the
```

Comment:
This method was added by @SimonRichardson too it seems.
Resolved conflict appears to move it needlessly.